setup.py

```diff
@@ -3,14 +3,18 @@
 from setuptools import find_packages, setup
 
 path_to_py42_wheel = os.path.join(os.getcwd(), 'wheels', 'py42-1.14.2-py2.py3-none-any.whl')
+path_to_stubs = os.path.join(os.getcwd(), 'stubs#egg=phantom')
 
 setup(
-    name="phcode42v2",
+    name="code42_connector",
     version="1.0.0",
     description="Code42 for Phantom",
-    packages=find_packages(),
-    install_requires=["requests>=2.3",
```

Comment:
How come we are removing `requests`?  It is imported in `code42_connector.py`

Reply:
Also (from other comment but means more here):

I noticed that requests on the server us up to version 2.24.0.
Should we make that is minimally the version we use?

Reply:
This was a mistake, `requests` should be here and not in the stubs